src/main/java/core/basesyntax/FigureInfoInterface.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface FigureInfoInterface {
+
+    public abstract String getFigureInfo();
+}

```

Comment:
```suggestion
public interface Drawer {
    void draw();
}

```

We don't need to use "Figure" word in interface or it's method name as we can use this interface not only for Figure classes. And all methods without implementation in interfaces are "public abstract" by default. And this method should be void(task)

Reply:
public abstract -  це написав свідомо, щоб зарубцювалось в пам'яті.